packages/react-router/src/ReactRouter/IonRouteInner.tsx

```diff
@@ -9,6 +9,12 @@ export class IonRouteInner extends React.PureComponent<IonRouteProps> {
         path={this.props.path}
         exact={this.props.exact}
         render={this.props.render}
+        /**
+         * `computedMatch` is a private API in react-router v5 that
+         * has been removed in v6.
+         *
+         * This needs to be removed when we support v6.
+         */
         computedMatch={(this.props as any).computedMatch}
```

Comment:
Do we have a Jira ticket for this yet?

Reply:
Good call, creating a spike for this now. Will update the comment to include the task 👍